pkg/credentials/credentials.go

```diff
@@ -159,6 +163,16 @@ func getKubeVirtPluginImage(dpa *oadpv1alpha1.DataProtectionApplication) string
 	return os.Getenv("RELATED_IMAGE_KUBEVIRT_VELERO_PLUGIN")
 }
 
+func getHypershiftPluginImage(dpa *oadpv1alpha1.DataProtectionApplication) string {
+	if dpa.Spec.UnsupportedOverrides[oadpv1alpha1.HypershiftPluginImageKey] != "" {
+		return dpa.Spec.UnsupportedOverrides[oadpv1alpha1.HypershiftPluginImageKey]
+	}
+	if os.Getenv("RELATED_IMAGE_HYPERSHIFT_VELERO_PLUGIN") == "" {
+		return common.HypershiftPluginImage
+	}
+	return os.Getenv("RELATED_IMAGE_HYPERSHIFT_VELERO_PLUGIN")
```

Comment:
same comment for all the places.

Reply:
@shubham-pampattiwar , I would rather instead advocate to change the plugin repo name. :pleading_face: 

A lot of downstream work has been completed based on [`oadp-hypershift-velero-plugin-container`](https://redhat-internal.slack.com/archives/C0144ECKUJ0/p1741773839662159?thread_ts=1741625011.483529&cid=C0144ECKUJ0) proposed in Slack (Mar 12th) without objections and subsequently discussed in Scrum (Mar 13th) with consensus for the name which was then fed back ([for example](https://github.com/openshift/oadp-operator/pull/1676#issuecomment-2741460086)) to ongoing efforts in github -- primarily, this PR.  :sweat_smile:

Reply:
I was thinking that both (dowstream images and the repo name with upstream images) were fully separated, is that the case? should I rename all the things based on `oadp-hypershift-velero-plugin-container`?

Reply:
They are indeed distinct & separate items.  The only real concern is getting the plumbing correct, so we override/inject/replace appropriately as things make their way from upstream through midstream and land in downstream to then produce official RH image build(s).  

As it stands, all the plumbing should be correct (as expected) with this PR merged.
I wouldn't worry too much about renaming anything at the moment while other, arguably more important, work is outstanding.  :sunglasses: